script.js

```diff
@@ -0,0 +1,171 @@
+const toDoForm = document.querySelector(".toDo__form"),
+  toDoInput = document.querySelector("input"),
+  toDoWaitingItems = document.querySelector(".waiting__items"),
+  toDoFinishedItems = document.querySelector(".finished__items"),
+  toDoInputIcon = document.querySelector(".input__icon"),
+  waitingListCount = document.querySelector(".waiting__number"),
+  finishedListCount = document.querySelector(".finished__number");
+
+const TODO_LS = "toDos";
+
+//할일 목록을 담은 list
+let toDos = [];
+
+//리스트 별 할일의 개수를 센다.
+function updateCount() {
+  //wcnt : waitingList 할일 개수, fcnt : finishedList 할일 개수
+  let wcnt = 0,
+    fcnt = 0;
+  //리스트를 순환하며 개수를 센다.
+  toDos.forEach((toDo) => {
+    if (toDo.opt == 1) {
+      wcnt++;
+    } else {
+      fcnt++;
+    }
+  });
+  //개수를 웹페이지에 나타내기
+  waitingListCount.innerText = `(${wcnt})`;
+  finishedListCount.innerText = `(${fcnt})`;
```

Comment:
저도 비슷한 방식으로 할일의 개수를 구했어요! 그래서 시원님이 배열에서 `filter`메소드를 사용하는 더 좋은  방법을 알려주셨어요. 이런식으로 짜면 코드 두 줄이면 해결되더라구요 😂😂😂
```suggestion
  waitingListCount.innerText = `(${toDos.filter(item => (item.opt == 1)).length})`;
  finishedListCount.innerText = `(${toDos.filter(item => !(item.opt == 1)).length})`;
```

Reply:
리스트를 순회하면서 구하는 방법이 있었군요!! 덕분에 다른 방법 하나 더 알아갑니다!
저는 전역으로 변수를 둬 상황에 따라 +, -를 해줬습니다!

Reply:
와... 간결한 코드에 무릎을 탁 쳤습니다... 공유해주셔서 감사합니다 ㅠㅠ